docs/examples/009-subsetting.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>Maybe just one sentence about how we will access data, e.g. access with geocat-datafiles through its GH repo, and what data vwe will use for this example, e.g. relative humidity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>